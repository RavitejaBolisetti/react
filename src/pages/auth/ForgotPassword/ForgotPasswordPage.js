import React from 'react';
import { ForgotPassword } from 'components/Auth';

export default function ForgotPasswordPage() {
    return (
        <div>
            <ForgotPassword />
        </div>
    );
}
