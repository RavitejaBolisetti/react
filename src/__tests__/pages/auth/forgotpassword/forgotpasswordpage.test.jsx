import '@testing-library/jest-dom/extend-expect';
import customRender from '@utils/test-utils';
import ForgotPasswordPage from '@pages/auth/ForgotPassword/ForgotPasswordPage';

describe('ForgotPasswordPageBase Components', () => {
    it('should render ForgotPasswordPageBase components', () => {
        const ForgotpasswordPage = customRender(<ForgotPasswordPage />);
        expect(ForgotpasswordPage).toMatchSnapshot();
    });
});
