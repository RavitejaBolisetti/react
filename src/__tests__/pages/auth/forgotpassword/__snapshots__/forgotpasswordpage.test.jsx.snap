// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForgotPasswordPageBase Components should render ForgotPasswordPageBase components 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="loginSection"
        >
          <div
            class="loginMnMlogo"
          >
            <img
              alt=""
              src="test-file-stub"
            />
          </div>
          <div
            class="center"
          >
            <div
              class="loginLogoSection"
            >
              <img
                alt=""
                class="mainLogo"
                src="test-file-stub"
              />
              <br />
              <img
                alt=""
                class="mainLogoLine"
                src="test-file-stub"
              />
              <div
                class="logoText"
              >
                Dealer Management System
              </div>
            </div>
            <div
              class="loginWrap"
            >
              <div
                class="ant-row css-dev-only-do-not-override-14wwjjs"
              >
                <div
                  class="ant-col ant-col-24 css-dev-only-do-not-override-14wwjjs"
                >
                  <div
                    class="loginHtml"
                  >
                    <div
                      class="centerInner"
                    >
                      <div
                        class="loginForm"
                      >
                        <form
                          autocomplete="off"
                          class="ant-form ant-form-horizontal css-dev-only-do-not-override-14wwjjs"
                          id="verifyUser"
                        >
                          <div
                            class="loginHeading"
                          >
                            <h1>
                              Forgot Your Password
                            </h1>
                            <div
                              class="loginSubHeading"
                            >
                              Please enter your user credential
                            </div>
                          </div>
                          <div
                            class="ant-row css-dev-only-do-not-override-14wwjjs"
                            style="margin-left: -10px; margin-right: -10px;"
                          >
                            <div
                              class="ant-col textfieldWithPrefix ant-col-xs-24 ant-col-sm-24 ant-col-md-24 ant-col-lg-24 ant-col-xl-24 css-dev-only-do-not-override-14wwjjs"
                              style="padding-left: 10px; padding-right: 10px;"
                            >
                              <div
                                class="ant-form-item inputBox marginBottomZero css-dev-only-do-not-override-14wwjjs"
                              >
                                <div
                                  class="ant-row ant-form-item-row css-dev-only-do-not-override-14wwjjs"
                                  data-testid="userIdInput"
                                >
                                  <div
                                    class="ant-col ant-form-item-control css-dev-only-do-not-override-14wwjjs"
                                  >
                                    <div
                                      class="ant-form-item-control-input"
                                    >
                                      <div
                                        class="ant-form-item-control-input-content"
                                      >
                                        <span
                                          class="ant-input-affix-wrapper css-dev-only-do-not-override-14wwjjs"
                                        >
                                          <span
                                            class="ant-input-prefix"
                                          >
                                            <svg
                                              fill="currentColor"
                                              height="18"
                                              stroke="currentColor"
                                              stroke-width="0"
                                              viewBox="0 0 24 24"
                                              width="18"
                                              xmlns="http://www.w3.org/2000/svg"
                                            >
                                              <path
                                                d="M12 2a5 5 0 1 0 5 5 5 5 0 0 0-5-5zm0 8a3 3 0 1 1 3-3 3 3 0 0 1-3 3zm9 11v-1a7 7 0 0 0-7-7h-4a7 7 0 0 0-7 7v1h2v-1a5 5 0 0 1 5-5h4a5 5 0 0 1 5 5v1z"
                                              />
                                            </svg>
                                          </span>
                                          <input
                                            aria-required="true"
                                            class="ant-input css-dev-only-do-not-override-14wwjjs"
                                            id="userId"
                                            maxlength="25"
                                            type="text"
                                            value=""
                                          />
                                        </span>
                                      </div>
                                    </div>
                                  </div>
                                </div>
                              </div>
                              <label
                                class="textfieldWithPrefix__label"
                              >
                                User ID (MILE ID.Parent ID)
                              </label>
                            </div>
                          </div>
                          <div
                            class="ant-row css-dev-only-do-not-override-14wwjjs"
                            style="margin-left: -10px; margin-right: -10px;"
                          >
                            <div
                              class="ant-col ant-col-xs-24 ant-col-sm-24 ant-col-md-24 ant-col-lg-24 ant-col-xl-24 css-dev-only-do-not-override-14wwjjs"
                              style="padding-left: 10px; padding-right: 10px;"
                            >
                              <button
                                class="ant-btn css-dev-only-do-not-override-14wwjjs ant-btn-primary button"
                                form="verifyUser"
                                type="submit"
                              >
                                <span
                                  class="ant-btn-icon"
                                >
                                  <svg
                                    fill="none"
                                    height="18"
                                    stroke="currentColor"
                                    stroke-linecap="round"
                                    stroke-linejoin="round"
                                    stroke-width="2"
                                    viewBox="0 0 24 24"
                                    width="18"
                                    xmlns="http://www.w3.org/2000/svg"
                                  >
                                    <rect
                                      height="11"
                                      rx="2"
                                      ry="2"
                                      width="18"
                                      x="3"
                                      y="11"
                                    />
                                    <path
                                      d="M7 11V7a5 5 0 0 1 10 0v4"
                                    />
                                  </svg>
                                </span>
                                <span>
                                  Verify User
                                </span>
                              </button>
                            </div>
                          </div>
                          <div
                            class="ant-row css-dev-only-do-not-override-14wwjjs"
                            style="margin-left: -10px; margin-right: -10px;"
                          >
                            <div
                              class="ant-col ant-col-xs-24 ant-col-sm-24 ant-col-md-24 ant-col-lg-24 ant-col-xl-24 css-dev-only-do-not-override-14wwjjs"
                              style="padding-left: 10px; padding-right: 10px;"
                            >
                              <div
                                class="loginFooter"
                                type="radio"
                              >
                                <a
                                  href="/login"
                                >
                                  Back to Login
                                </a>
                              </div>
                            </div>
                          </div>
                        </form>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="loginMainFooter"
          >
            <div
              class="ant-row css-dev-only-do-not-override-14wwjjs"
              style="margin-left: -10px; margin-right: -10px;"
            >
              <div
                class="ant-col ant-col-xs-24 ant-col-sm-24 ant-col-md-24 ant-col-lg-24 ant-col-xl-24 css-dev-only-do-not-override-14wwjjs"
                style="padding-left: 10px; padding-right: 10px;"
              >
                <div
                  class="footerText"
                >
                  <span>
                    Copyright © 
                    2023 
                     ROBIN.
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      ;
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="loginSection"
      >
        <div
          class="loginMnMlogo"
        >
          <img
            alt=""
            src="test-file-stub"
          />
        </div>
        <div
          class="center"
        >
          <div
            class="loginLogoSection"
          >
            <img
              alt=""
              class="mainLogo"
              src="test-file-stub"
            />
            <br />
            <img
              alt=""
              class="mainLogoLine"
              src="test-file-stub"
            />
            <div
              class="logoText"
            >
              Dealer Management System
            </div>
          </div>
          <div
            class="loginWrap"
          >
            <div
              class="ant-row css-dev-only-do-not-override-14wwjjs"
            >
              <div
                class="ant-col ant-col-24 css-dev-only-do-not-override-14wwjjs"
              >
                <div
                  class="loginHtml"
                >
                  <div
                    class="centerInner"
                  >
                    <div
                      class="loginForm"
                    >
                      <form
                        autocomplete="off"
                        class="ant-form ant-form-horizontal css-dev-only-do-not-override-14wwjjs"
                        id="verifyUser"
                      >
                        <div
                          class="loginHeading"
                        >
                          <h1>
                            Forgot Your Password
                          </h1>
                          <div
                            class="loginSubHeading"
                          >
                            Please enter your user credential
                          </div>
                        </div>
                        <div
                          class="ant-row css-dev-only-do-not-override-14wwjjs"
                          style="margin-left: -10px; margin-right: -10px;"
                        >
                          <div
                            class="ant-col textfieldWithPrefix ant-col-xs-24 ant-col-sm-24 ant-col-md-24 ant-col-lg-24 ant-col-xl-24 css-dev-only-do-not-override-14wwjjs"
                            style="padding-left: 10px; padding-right: 10px;"
                          >
                            <div
                              class="ant-form-item inputBox marginBottomZero css-dev-only-do-not-override-14wwjjs"
                            >
                              <div
                                class="ant-row ant-form-item-row css-dev-only-do-not-override-14wwjjs"
                                data-testid="userIdInput"
                              >
                                <div
                                  class="ant-col ant-form-item-control css-dev-only-do-not-override-14wwjjs"
                                >
                                  <div
                                    class="ant-form-item-control-input"
                                  >
                                    <div
                                      class="ant-form-item-control-input-content"
                                    >
                                      <span
                                        class="ant-input-affix-wrapper css-dev-only-do-not-override-14wwjjs"
                                      >
                                        <span
                                          class="ant-input-prefix"
                                        >
                                          <svg
                                            fill="currentColor"
                                            height="18"
                                            stroke="currentColor"
                                            stroke-width="0"
                                            viewBox="0 0 24 24"
                                            width="18"
                                            xmlns="http://www.w3.org/2000/svg"
                                          >
                                            <path
                                              d="M12 2a5 5 0 1 0 5 5 5 5 0 0 0-5-5zm0 8a3 3 0 1 1 3-3 3 3 0 0 1-3 3zm9 11v-1a7 7 0 0 0-7-7h-4a7 7 0 0 0-7 7v1h2v-1a5 5 0 0 1 5-5h4a5 5 0 0 1 5 5v1z"
                                            />
                                          </svg>
                                        </span>
                                        <input
                                          aria-required="true"
                                          class="ant-input css-dev-only-do-not-override-14wwjjs"
                                          id="userId"
                                          maxlength="25"
                                          type="text"
                                          value=""
                                        />
                                      </span>
                                    </div>
                                  </div>
                                </div>
                              </div>
                            </div>
                            <label
                              class="textfieldWithPrefix__label"
                            >
                              User ID (MILE ID.Parent ID)
                            </label>
                          </div>
                        </div>
                        <div
                          class="ant-row css-dev-only-do-not-override-14wwjjs"
                          style="margin-left: -10px; margin-right: -10px;"
                        >
                          <div
                            class="ant-col ant-col-xs-24 ant-col-sm-24 ant-col-md-24 ant-col-lg-24 ant-col-xl-24 css-dev-only-do-not-override-14wwjjs"
                            style="padding-left: 10px; padding-right: 10px;"
                          >
                            <button
                              class="ant-btn css-dev-only-do-not-override-14wwjjs ant-btn-primary button"
                              form="verifyUser"
                              type="submit"
                            >
                              <span
                                class="ant-btn-icon"
                              >
                                <svg
                                  fill="none"
                                  height="18"
                                  stroke="currentColor"
                                  stroke-linecap="round"
                                  stroke-linejoin="round"
                                  stroke-width="2"
                                  viewBox="0 0 24 24"
                                  width="18"
                                  xmlns="http://www.w3.org/2000/svg"
                                >
                                  <rect
                                    height="11"
                                    rx="2"
                                    ry="2"
                                    width="18"
                                    x="3"
                                    y="11"
                                  />
                                  <path
                                    d="M7 11V7a5 5 0 0 1 10 0v4"
                                  />
                                </svg>
                              </span>
                              <span>
                                Verify User
                              </span>
                            </button>
                          </div>
                        </div>
                        <div
                          class="ant-row css-dev-only-do-not-override-14wwjjs"
                          style="margin-left: -10px; margin-right: -10px;"
                        >
                          <div
                            class="ant-col ant-col-xs-24 ant-col-sm-24 ant-col-md-24 ant-col-lg-24 ant-col-xl-24 css-dev-only-do-not-override-14wwjjs"
                            style="padding-left: 10px; padding-right: 10px;"
                          >
                            <div
                              class="loginFooter"
                              type="radio"
                            >
                              <a
                                href="/login"
                              >
                                Back to Login
                              </a>
                            </div>
                          </div>
                        </div>
                      </form>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="loginMainFooter"
        >
          <div
            class="ant-row css-dev-only-do-not-override-14wwjjs"
            style="margin-left: -10px; margin-right: -10px;"
          >
            <div
              class="ant-col ant-col-xs-24 ant-col-sm-24 ant-col-md-24 ant-col-lg-24 ant-col-xl-24 css-dev-only-do-not-override-14wwjjs"
              style="padding-left: 10px; padding-right: 10px;"
            >
              <div
                class="footerText"
              >
                <span>
                  Copyright © 
                  2023 
                   ROBIN.
                </span>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    ;
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
